docs/_extra/api-reference/hypothesis.yaml

```diff
@@ -443,6 +443,38 @@ paths:
           description: Group not found
           schema:
             $ref: '#/definitions/Error'
+    post:
+      tags:
+        - groups
+      summary: Add a member to a group
+      operationId: addGroupMember
+      parameters:
+        - name: id
+          in: path
+          description: Public ID of the group
+          required: true
+          type: string
+        - name: user
+          in: path
+          description: |
+            User ID of the user to add to the group.
+          required: true
+          type: string
+      responses:
+        '204':
+          description: Success
+        '404':
+          description: Group not found
+          schema:
+            $ref: '#/definitions/Error'
+        '404':
+          description: User not found
+          schema:
+            $ref: '#/definitions/Error'
+        '404':
```

Comment:
This is the correct HTTP response to return in this situation (see another comment I made) but it should not disclose the authority mismatch or it's leaking privileged information about groups or users belonging to other authorities.